File: core/src/org/rexellentgames/dungeon/debug/DieCommand.java
package org.rexellentgames.dungeon.debug;

import org.rexellentgames.dungeon.entity.creature.player.Player;

public class DieCommand extends ConsoleCommand {
	{
		shortName = "/de";
		name = "/die";
		description = "Death";
	}

	@Override
	public void run(Console console, String[] args) {
		if (Player.instance != null) {
			Player.instance.die();
		}
	}
}

File: core/src/org/rexellentgames/dungeon/debug/DieCommand.kt
package org.rexellentgames.dungeon.debug

import org.rexellentgames.dungeon.entity.creature.player.Player

class DieCommand : ConsoleCommand("/die", "de", "Death") {
    override fun run(console: Console, args: Array<String>) {
        if (Player.instance != null) {
            Player.instance.die()
        }
    }
}